src/lib.rs

```diff
@@ -88,12 +100,28 @@ impl Sdk {
         tracing::info!("our peer id is: {}", peer);
 
         let transport = transport::transport(keypair.to_libp2p())?;
+        let listen_on = if cfg!(target_family = "wasm") {
+            "/dns4/local1st.net/tcp/443/wss/p2p-webrtc-star"
+                .parse()
+                .unwrap()
+
+            //TODO
+            //        slf.add_external_address(
+            //            signaling_server
+            //                .with(Protocol::P2pWebRtcStar)
+            //                .with(Protocol::P2p(libp2p_peer.into())),
+            //            // TODO
+            //            AddressScore::Infinite,
+            //        )
```

Comment:
this needed or not?

Reply:
Not sure yet. Will look into the actual libp2p connection behaviour next.